src/html/entrypoint.pug

```diff
@@ -54,57 +54,57 @@ html
+                            input#auth_username.form-control(type='text', placeholder='Enter your username for the SMTP server specified above.', value='#{config.email_account.username}')
                     .form-group
                         label.control-label.col-sm-2(for='auth_password') Password:
                         .col-sm-10
-                            input#auth_password.form-control(type='password', placeholder='Enter the password corresponding to your username above.')
+                            input#auth_password.form-control(type='password', placeholder='Enter the password corresponding to your username above.', value='#{config.email_account.password}')
             #tab_send
                 form.form-horizontal
                     .form-group
                         label.control-label.col-sm-2(for='send_from_name') Sender name:
                         .col-sm-10
-                            input#send_from_name.form-control(type='text', value="NUS Students' Computing Club Management Committee")
+                            input#send_from_name.form-control(type='text', value='#{config.email_header.sender_name}')
                     .form-group
                         label.control-label.col-sm-2(for='send_from_email') Sender email address:
                         .col-sm-10
-                            input#send_from_email.form-control(type='email', placeholder='Enter the email address to blast from.')
+                            input#send_from_email.form-control(type='email', placeholder='Enter the email address to blast from.', value='#{config.email_header.sender_email}')
                     .form-group
                         label.control-label.col-sm-2(for='send_to_name') Recipient name:
                         .col-sm-10
-                            input#send_to_name.form-control(type='text', value="NUS Students' Computing Club Members")
+                            input#send_to_name.form-control(type='text', value='#{config.email_header.recipient_name}')
                     .form-group
                         label.control-label.col-sm-2(for='send_to_email') Recipient email address:
                         .col-sm-10
-                            input#send_to_email.form-control(type='email', placeholder='Enter the email address to blast to.', value='undergrad-l@comp.nus.edu.sg')
+                            input#send_to_email.form-control(type='email', placeholder='Enter the email address to blast to.', value='#{config.email_header.sender_email}')
                     .form-group
                         label.control-label.col-sm-2(for='send_bcc') BCC:
                         .col-sm-10
-                            input#send_bcc.form-control(type='email', value='marketing@nuscomputing.com')
+                            input#send_bcc.form-control(type='email', value='#{config.email_header.bcc}')
```

Comment:
```javascript
value=`${config.email_header.bcc}`
```

Use ES6 interpolation syntax here. I don't think single quotes does any interpolation (at least I don't see it in the Pug docs).